indymeet/settings/production.py

```diff
@@ -55,10 +55,7 @@
     STATIC_URL = f"https://{AZURE_CUSTOM_DOMAIN}/{AZURE_STATIC_CONTAINER}/"
     MEDIA_URL = f"https://{AZURE_CUSTOM_DOMAIN}/{AZURE_MEDIA_CONTAINER}/"
 
-    SENTRY_DNS = os.environ.get(
-        "SENTRY_DNS",
-        "https://302dde9a33fabc5af106ccf19ae23680@o4506747118157824.ingest.sentry.io/4506747129626624"  # Staging
-    )
+    SENTRY_DNS = os.environ.get("SENTRY_DNS")
```

Comment:
I created a new DSN for staging and added it as an environment variable to our azure staging environment. When you merge this, I'll delete the DSN we had previously.